### ISSUE TITLE
Don't call a promise a dataSource

### ISSUE THREAD

**Issue by emackey** [AnalyticalGraphicsInc/cesium#5192]
Several of our examples have [code that looks like this](https://github.com/AnalyticalGraphicsInc/cesium/blob/a81f2e7de64c2348558046b5e8e685278fde0dcf/Apps/Sandcastle/gallery/CZML%20Point.html#L55-L57):

```
var dataSource = Cesium.CzmlDataSource.load(czml);
viewer.dataSources.add(dataSource);
viewer.zoomTo(dataSource);
```

This code gets a *promise* to a data source, and stores it in a variable named `dataSource`.  The `add` and `zoomTo` methods are both equipped to accept a promise, but more generic actions such as attempting to access `dataSource.entities` will fail.

I propose we find all these examples, and rename `dataSource` to `dataSourcePromise` everywhere it is appropriate.

**Comment by naeri-kailash** [comment]
I will take a look at this issue if that's okay

**Comment by emackey** [comment]
Fixed in #5205.
